rr/gstreamer/gst_media.py

```diff
@@ -10,6 +10,28 @@
 from gi.repository import Gst as gst  # nopep8
 from gi.repository import GLib  # nopep8
 
+import numpy as np
+import time
```

Comment:
What include ordering are you following?

PEP8 is:
* Standard library imports.
* Related third party imports.
* Local application/library specific imports.